get-git-repos/download_repos.py

```diff
@@ -1,28 +1,57 @@
-#! /usr/bin/python
+#!/usr/bin/env python3
 
 import os
 
 # TODO: add options to include debian, arch, and/or fedora packages and remove them from PKGS.
 
 # MATE repos
-PKGS = ["mate-user-share", "debian-packages", "mate-bluetooth", "mate-panel", "mate-notification",
-        "mate-keyring", "mate-indicator-applet", "libmatekeyring", "mate-window-manager",
-        "mate-session-manager", "libmatecomponent", "mate-themes", "mate-file-manager", 
-        "mate-system-tools", "fedora-packages", "archlinux-packages", "mate-common",
-        "mate-character-map", "Mate-Extra", "mate-control-center", "mate-settings-daemon",
-        "caja-dropbox", "mate-file-archiver", "mate-utils", "mate-applets", "mate-media",
-        "mate-desktop", "mate-calc", "mate-image-viewer", "mate-file-manager-open-terminal",
-        "libmatekbd", "mate-power-manager", "mate-system-monitor", "mate-text-editor", 
-        "mate-terminal", "mate-document-viewer", "mate-file-manager-gksu", "mate-mime-data",
-        "mate-vfs", "libmate", "mate-backgrounds", "mate-corba", "mate-file-manager-image-converter",
-        "mate-file-manager-sendto", "mate-menu-editor", "python-caja", "mate-conf-editor",
-        "ffmpegthumbnailer-caja", "mate-sensors-applet", "mate-netspeed", "python-mate", 
-        "python-corba", "mate-doc-utils", "mate-screensaver", "mate-polkit", "mate-menus",
-        "mate-dialogs", "mate-icon-theme", "libmateweather", "libmateui", "mate-conf",
-        "libmatecomponentui", "libmatecanvas", "mate-file-manager-share", "python-mate-desktop"]
+PKGS = ["atril",
+        "caja",
+        "caja-dropbox",
+        "caja-extensions",
+        "caja-xattrs",
+        "debian-packages",
+        "engrampa",
+        "eom",
+        "libmatekbd",
+        "libmatemixer",
+        "libmateweather",
+        "marco",
+        "mate-applets",
+        "mate-backgrounds",
+        "mate-calc",
+        "mate-common",
+        "mate-control-center",
+        "mate-desktop",
+        "mate-desktop.org",
+        "mate-dev-scripts",
+        "mate-icon-theme",
+        "mate-icon-theme-faenza",
+        "mate-indicator-applet",
+        "mate-media",
+        "mate-menus",
+        "mate-netbook",
+        "mate-notification-daemon",
+        "mate-panel",
+        "mate-polkit",
+        "mate-power-manager",
+        "mate-screensaver",
+        "mate-sensors-applet",
+        "mate-session-manager",
+        "mate-settings-daemon",
+        "mate-system-monitor",
+        "mate-terminal",
+        "mate-themes",
+        "mate-university",
+        "mate-user-guide",
+        "mate-user-share",
+        "mate-utils",
+        "mozo",
+        "pluma",
+        "python-caja"]
       
-GIT_SSH = "git clone git@github.com:mate-desktop/"
+GIT_SSH = "git clone https://github.com/mate-desktop/"
```

Comment:
I use an SSH keypair for my repos, changing this to HTTPS would break things for me and drive me crazy (I don't even remember what my GitHub password is for that matter...). However, I haven't actually used this script - I just clone my repos manually